database/migrations/2021_05_25_134101_create_gadgets_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateGadgetsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('gadgets', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->timestamps();
```

Comment:
```suggestion
```
Waarvoor worden deze `timestamps` gebruikt? Volgensmij zijn deze optioneel.

Reply:
Voor de andere tables hebben we ook standaard timestamps welke niet gebruikt worden (zoals bij loot als voorbeeld). We houden dus meer de standaard aan door ze hier ook te hebben.

Reply:
P.S: vorige comment leest een beetje raar: bedoelde meer dat iedere table ze heeft, en sommige hebben ook niet per se een nut. Sommige tables maken er wel specifiek gebruik van.